src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -8948,6 +9225,38 @@ export class StudyViewPageStore
         } else return '';
     }
 
+    public async getMutationTypesDownloadData(
```

Comment:
move to util file

Reply:
I also updated similar download functions.